front/lib/api/assistant/conversation.ts

```diff
@@ -143,18 +149,22 @@ export async function updateConversation(
     throw new Error(`Conversation ${conversationId} not found`);
   }
 
+  if (!(await canAccessConversation(auth, conversation))) {
+    throw new Error("Conversation access denied");
```

Comment:
Is this really a throw here? A throw is a panic.

Reply:
Following the convention for that function, just above you can see a throw for conversation not found; in both cases it's really not gonna work if you try to update the conversation, and should basically never happen

We could refactor the function to return a result, but the PR being big I thought best to leave it oustide scope